src/flamenco/runtime/tests/fd_exec_instr_test.c

```diff
@@ -159,10 +154,8 @@ static int
 _context_create( fd_exec_instr_test_runner_t *        runner,
                  fd_exec_instr_ctx_t *                ctx,
                  fd_exec_test_instr_context_t const * test_ctx,
+                 fd_alloc_t *                         alloc,
```

Comment:
We should probably pass a `fd_valloc_t` so we can easily swap this out in future.